### PR TITLE
Feature/aich interface

### PR DESCRIPTION
**Pull request by TinyMurky**
### DEVELOP
- 更改IInvoice, IVoucher, IPayment 的type
- /Users/tinymurky/Documents/mycode/iSunFA/src/components/new_journal_form/new_journal_form.tsx 內部有 `To Emily, TO Julian`字樣的是被修改的code
- /Users/tinymurky/Documents/mycode/iSunFA/src/interfaces/account.ts 有新的EventType, VoucherType,  PaymentStatusType, PaymentPeriodType
- api回傳暫時可以看 [點我](https://documenter.getpostman.com/view/29158306/2sA3JQ4f6q)

### CHECK LIST
- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 
- new Class / Component: 
 - new loop: 0
- new recursive: 0
- high risk: 0
- new sql: 0

### CODE REVIEW

File: src/interfaces/accounting_account.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { ProgressStatus, isProgressStatus } from './common';
+import { ProgressStatus, isProgressStatus } from './account';
```
Comment (Luphia) [inline src/interfaces/accounting_account.ts:1]: - use @ for import path

File: src/interfaces/invoice.ts
Comment (Luphia) [inline src/interfaces/invoice.ts:1]: - use @ for import path

File: src/interfaces/account.ts
Comment (Luphia) [inline src/interfaces/account.ts:1]: - declare enum as a constant

**Review by Luphia — CHANGES_REQUESTED** [review]

File: src/interfaces/invoice.ts
Comment (TzuHanLiang) [inline src/interfaces/invoice.ts:1]: import { EventType, isEventType } from '@/interfaces/account'; 要改成
import { EventType, isEventType } from '@/constants/account';

File: src/constants/account.ts
Comment (Luphia) [inline src/constants/account.ts:1]: - use SNAKE_CASE for constant key

**Review by Luphia — CHANGES_REQUESTED** [review]

File: src/interfaces/invoice.ts
Comment (Luphia) [reply src/interfaces/invoice.ts:1]: - Fixed with [b130d1a](https://github.com/CAFECA-IO/iSunFA/pull/678/commits/b130d1af68fdeb3ab4abf592ed919ee6e8239904)

File: src/constants/account.ts
Comment (Luphia) [reply src/constants/account.ts:1]: - ToDo later

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**